### ISSUE TITLE
IPv6 support

### ISSUE THREAD

**Issue by tambry** [browsh-org/www.brow.sh#4]
The www.brow.sh website seems to be inaccessible using IPv6.

**Comment by tombh** [comment]
Wow! I've never setup IPv6 before, feels good :D 

I added them just this second, so hopefully they'll propagate soon.

**Comment by tambry** [comment]
Can confirm www.brow.sh as fixed. Thanks for taking a look at this! 👍 

As a small note, brow.sh also lacks IPv6 support. It seems to be a GCP IP, so there's probably nothing that can be done about that for now.

**Comment by tombh** [comment]
brow.sh, html.brow.sh and text.brow.sh are all actually GCP load balancers, which, as I've just read, will indeed accept static IPv6 addresses. When I get a moment I'll see what I can do. I want to make sure I can still associate reserved IPv6 addresses in my Terraform setup. Last time I tried associating the IPv4 addresses I deleted the reserved addresses and brought the site down. So I gotta be in the right mood, if you know what I mean, haha.

**Comment by tombh** [comment]
brow.sh, html.brow.sh and text.brow.sh are all actually GCP load balancers, which, as I've just read, will indeed accept static IPv6 addresses. When I get a moment I'll see what I can do. I want to make sure I can still associate reserved IPv6 addresses in my Terraform setup. Last time I tried associating the IPv4 addresses I deleted the reserved addresses and brought the site down. So I gotta be in the right mood, if you know what I mean, haha.

Edit: browsh-org/baas#1
